### ISSUE TITLE
PieChart strokeLinejoin doesnt work

### ISSUE THREAD

**Issue by Knight704** [JesperLekland/react-native-svg-charts#263]
### What is the problem?
strokeLinejoin: 'round' doesnt do rounded corners to Pie slices

### When does it happen?
Using PieChart component

### What platform?
iOS, Android

React Native version: 0.57

### Code to reproduce

```javascript

const data = [
      347,
      500,
      153,
    ].map((value, index) => ({
      value,
      key: `slise-${index}`,
      svg: {
        strokeLinejoin: 'round',
        fill: COLORS[index],
      },
    }))

<PieChart
          data={data}
          startAngle={-Math.PI / 2}
          endAngle={Math.PI / 2}
          padAngle={0}
          outerRadius="100%"
          innerRadius="88%"
        />
```

Result: no rounded corners
![screen shot 2018-11-09 at 17 19 45](https://user-images.githubusercontent.com/2227516/48264645-aa8b5580-e443-11e8-84ca-1da7775f497b.png)

**Comment by Knight704** [comment]
using padRadius inside ```pie-chart.js``` as part of d3-shape construction provide nearly expected behavior, except it also round corners between slices (not only at start and end)

**Comment by Knight704** [comment]
[hmmm, SO answer related to d3 lib](https://stackoverflow.com/questions/40445033/round-one-side-of-d3-arc)

**Comment by hitmanmcc** [comment]
Yeah, I bumped into this as well. I would like to only round one of the sides of the arcs, namely the left side of the first arc and the right side of the last arc. Did you find any workaround for this @Knight704 ? Or perhaps can you suggest something that would create this effect @JesperLekland?
Thanks.

**Comment by codeapp17** [comment]
@hitmanmcc Do you able to create the desired effect? I also needs to achieve same.

**Comment by hitmanmcc** [comment]
@codeapp17 No, we ended up dropping the idea and going a different route, sorry.

**Comment by codeapp17** [comment]
> @codeapp17 No, we ended up dropping the idea and going a different route, sorry.

No Problem, thanks anyways
